### ISSUE TITLE
ANDI: Element has no accessible name

### ISSUE THREAD

**Issue by JenM00re** [Swarthmore/accessible-docs#18]
Looks like the light/dark mode button has no accessible name.

![ANDI Testing No Accessible Name](https://github.com/Swarthmore/a11y-doc-repo/assets/91497451/3e7c9cd6-218a-4fc3-a215-e59fce791f9f)

Explanation:

- [Alert: No Accessible Name (General)](https://www.ssa.gov/accessibility/andi/help/alerts.html?no_name_generic)
- [Providing Accessible Names and Descriptions](https://www.w3.org/WAI/ARIA/apg/practices/names-and-descriptions/)
- [Accessible name | MDN Web Docs Glossary: Definitions of Web-related terms
](https://developer.mozilla.org/en-US/docs/Glossary/Accessible_name)
